src/CollAction/Models/Project.cs

```diff
@@ -67,11 +67,32 @@ public bool IsClosed
 
         public List<ProjectTag> Tags { get; set; }
         public List<ProjectParticipant> Participants { get; set; }
-
+        
         [NotMapped]
         public string HashTags
             => string.Join(";", Tags?.Select(tag => tag.Tag.Name) ?? Enumerable.Empty<string>());
 
+        [NotMapped]
+        public int RemainingDays
+            => Convert.ToInt32(Math.Round((End - Start).TotalDays));
+
+        [NotMapped]
+        [Display(Name = "Status")]
+        public string StatusDescription
```

Comment:
A more appropiate place would be to put it in the viewmodel. It's purely related to how to display a project, not the project data itself. It's also not localized. In fact, if you put this property here, it would also be impossible to localize in the future.